src/sbt-test/sbt-ensime/bad-build.scala/ensimeConfigProject.expect

```diff
@@ -19,5 +19,12 @@
0-sbt-2cc8d2761242b072cedb0a04cb39435c4fa24f9a.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/jansi-1.11.jar" "BASE_DIR/global/boot/scala-2.10.6/lib/jansi.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/jawn-parser_2.10-0.6.0.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/jline-2.13.jar" "BASE_DIR/global/boot/scala-2.10.6/lib/jline.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/jsch-0.1.46.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/json4s-ast_2.10-3.2.10.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/json4s-core_2.10-3.2.10.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/json4s-support_2.10-0.6.0.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/launcher-interface-1.0.0-M1.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/logging-0.13.12.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/logic-0.13.12.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/main-0.13.12.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/main-settings-0.13.12.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/paranamer-2.6.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/persist-0.13.12.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/process-0.13.12.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/quasiquotes_2.10-2.0.1.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/relation-0.13.12.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/run-0.13.12.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/sbinary_2.10-0.4.2.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/sbt-0.13.12.jar" "IVY_DIR/local/org.ensime/sbt-ensime/scala_2.10/sbt_0.13/HEAD/jars/sbt-ensime.jar" "BASE_DIR/global/boot/scala-2.10.6/lib/scala-compiler.jar" "BASE_DIR/global/boot/scala-2.10.6/lib/scala-library.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/scala-pickling_2.10-0.10.1.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/scala-reflect-2.10.6.jar" "BASE_DIR/global/boot/scala-2.10.6/lib/scala-reflect.jar" "IVY_DIR/cache/org.scala-lang/scalap/jars/scalap-2.10.6.jar" "IVY_DIR/cache/org.scalariform/scalariform_2.10/jars/scalariform_2.10-0.1.4.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/serialization_2.10-0.1.2.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/task-system-0.13.12.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/tasks-0.13.12.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/test-agent-0.13.12.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/test-interface-1.0.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/testing-0.13.12.jar" "BASE_DIR/global/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.12/tracking-0.13.12.jar")))))
```

Comment:
`deps` is confusing because in sbt it means inter project dependencies. Maybe `library-dependencies`? In genreal, let's try to follow the sbt conventions as closely as possible so it just makes everything simpler to understand.